core/variant/variant_call.cpp

```diff
@@ -2569,9 +2569,13 @@ static void _register_variant_builtin_methods() {
 
 	_VariantCall::add_variant_constant(Variant::VECTOR4I, "ZERO", Vector4i(0, 0, 0, 0));
 	_VariantCall::add_variant_constant(Variant::VECTOR4I, "ONE", Vector4i(1, 1, 1, 1));
+	_VariantCall::add_variant_constant(Variant::VECTOR4I, "MIN", Vector4i(INT_MIN, INT_MIN, INT_MIN, INT_MIN));
+	_VariantCall::add_variant_constant(Variant::VECTOR4I, "MAX", Vector4i(INT_MAX, INT_MAX, INT_MAX, INT_MAX));
```

Comment:
`Vector*i` explicitly use `int32_t` and not `int`, so this should be `INT32_MIN` / `INT32_MAX`.

`INT_MAX` varies based on the size of `int`, which depends on the system.